src/coreclr/inc/corinfoinstructionset.h

```diff
@@ -426,7 +426,7 @@ inline CORINFO_InstructionSetFlags EnsureInstructionSetFlagsAreValid(CORINFO_Ins
             resultflags.RemoveInstructionSet(InstructionSet_POPCNT);
         if (resultflags.HasInstructionSet(InstructionSet_Vector128) && !resultflags.HasInstructionSet(InstructionSet_SSE))
             resultflags.RemoveInstructionSet(InstructionSet_Vector128);
-        if (resultflags.HasInstructionSet(InstructionSet_Vector256) && !resultflags.HasInstructionSet(InstructionSet_AVX))
```

Comment:
This is the wrong way to modify this file, but also I wouldn't modify this file to get the desired behavior as it will prevent us from reporting `false` and still partially accelerating functions on AVX only hardware (among other issues). -- You also can't do this without potentially breaking ABI handling...

The "right" fix would be to special case `get_IsHardwareAccelerated` around  here: https://github.com/dotnet/runtime/blob/main/src/coreclr/jit/hwintrinsic.cpp#L288-L292

That way `Vector256` is treated as "supported" (this is important for interop/ABI handling) but `IsHardwareAccelerated` returns "false". You can then likewise force `Vector128` to report the same in the scenario a user disables `SSE2` (which just ensures we do the "right thing" even in the testing scenario).

Reply:
Got it, thanks!